src/store/AppStateStore.tsx

```diff
@@ -50,7 +49,7 @@ export default class AppStateStore extends React.Component<{}, IAppState> {
     );
   }
 
-  private validateDefinitionsJson(editor: IInstance, data: any, value: string) {
+  private validateDefinitionsJson(editor: any, data: any, value: string) {
     if(value.length > 0) {
```

Comment:
Surprised they did away with that interface entirely. We may want to update this signature to [match](https://github.com/scniro/react-codemirror2/blob/a633e7dd673ddf5bdb07e2ed664a03aa47159bfa/src/index.tsx#L44) the `ICodeMirror.onChange` signature at some point, but this will work for the purposes of this PR.